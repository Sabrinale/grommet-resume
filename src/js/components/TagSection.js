import React from 'react'

import { Text, Box } from 'grommet'

import { TagHeading } from './TagHeading'

export const TagSection = ({ tags, children }) => {
  if (tags) {
    return (
      <Box>
        <TagHeading>{children}</TagHeading>
        <Box direction="column" margin={{ top: '0px', left: 'xsmall' }}>
          {tags.map(item => (
            <Text key={item}>{item}</Text>
          ))}
        </Box>
      </Box>
    )
  }
  return <></>
}
