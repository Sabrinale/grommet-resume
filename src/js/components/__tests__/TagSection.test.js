import React from 'react'
import renderer from 'react-test-renderer'

import { TagSection } from '../TagSection'

test('renders tag heading element', () => {
  const component = renderer.create(<TagSection>test</TagSection>)
  let tree = component.toJSON()
  expect(tree).toMatchSnapshot()
})
