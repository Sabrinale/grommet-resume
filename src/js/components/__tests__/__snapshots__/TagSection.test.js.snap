// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders tag heading element 1`] = `null`;
